plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -2409,6 +2431,13 @@ def get_diff_merged(self, config):
                     self.result['response'] = self.msg
                     return self
 
+                regex_pattern = r'https://\S+'
+                if not re.match(regex_pattern, url):
+                    self.status = "failed"
+                    self.msg = "Given url '{0}' is invalid url for ITSM Intergartion setting. It must starts with 'https://'".format(url)
```

Comment:
It must **start** not **starts**.. 

`self.msg = "Given URL '{0}' is invalid for ITSM Integration settings. It must start with 'https://'.".format(url)"
`

Reply:
changed to start.